packages/react-super-components/src/translation/translations.ts

```diff
@@ -0,0 +1,17 @@
+import { Locale } from '@ovhcloud/msc-utils';
+
+export type Translations = {
+  see_more_label: string;
+};
+
+export async function getTranslations(locale?: Locale): Promise<Translations> {
+  try {
+    return await import(
+      `./translations/Messages_${locale?.replace('-', '_')}.json`
```

Comment:
You can't make a dynamic import with a variable inside it and you don't need a getTranslations actually